README.md

```diff
@@ -26,7 +26,7 @@ Features
     # Follow the instructions on screen
     $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
     ```
-1. Update your package database
+2. Update your package database
```

Comment:
I'm 85% sure we could put '1.' everywhere, and Markdown will figure out the proper sequence.

Reply:
@fmorency I had just assumed that the numbering was by accident, but it sounds like it shouldn't matter because of the way markdown is processed, so I'll revert this particular change.